src/Core/src/Platform/Android/MauiScrollView.cs

```diff
@@ -2,6 +2,7 @@
 using Android.Animation;
 using Android.Content;
 using Android.Graphics;
+using Android.Hardware.Lights;
```

Comment:
?

Reply:
this happens to me sometimes, is VS is adding this without us noticing ?